week1/practice/app/src/main/java/com/example/birthdaycard/MainActivity.kt

```diff
@@ -0,0 +1,87 @@
+package com.example.birthdaycard
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.layout.Arrangement.Top
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Alignment.Companion.CenterHorizontally
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.Role.Companion.Image
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.example.birthdaycard.ui.theme.BirthdayCardTheme
+
+class MainActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+            BirthdayCardTheme {
+                // A surface container using the 'background' color from the theme
+                Surface(
+                    modifier = Modifier.fillMaxSize(),
+                    color = MaterialTheme.colors.background
+                ) {
+                }
+            }
+        }
+    }
+}
+@Composable
+fun BirthdayGreetingWithText(message: String ,from : String,modifier: Modifier = Modifier) {
+    Column(
+        modifier=modifier.fillMaxSize(),
+        verticalArrangement = Top,
+        horizontalAlignment = CenterHorizontally,
+
+    ) {
+        Text(
+            text = message ,
+            fontSize = 36.sp,
+            modifier = Modifier.padding(
+                top = 16.dp,
+            )
+        )
+        Text(
+            text = from,
+            fontSize = 24.sp,
+            modifier = Modifier.padding(
+                top = 16.dp,
+            )
+
+        )
+    }
+}
+
+@Composable
+fun BirthdayGreetingWithImage(message: String ,from : String,modifier: Modifier = Modifier){
+    val image = painterResource(R.drawable.androidparty)
+    Box(){
```

Comment:
```suggestion
    Box(modifier = modifier) {
```

Box의 modifier를 지정해주는 것이 좋을 것 같습니다!